log/config.go

```diff
@@ -270,10 +279,22 @@ func Configure(options *Options) error {
 		return err
 	}
 
-	if err = updateScopes(options, core, errSink); err != nil {
+	if err = updateScopes(options); err != nil {
```

Comment:
Would it be simpler to follow the patchTable model for the global method overrides?
There are few public entry points. They can call private counter-parts with a patch table, similar to how Mixer server code works. WDYT?